clang/test/Driver/wasm-features.c

```diff
@@ -50,3 +60,73 @@
 // DEFAULT-NOT: "-target-feature" "-multimemory"
 // MVP-NOT: "-target-feature" "+multimemory"
 // BLEEDING-EDGE-NOT: "-target-feature" "-multimemory"
+
+// RUN: %clang --target=wasm32-unknown-unknown -### %s -matomics 2>&1 | FileCheck %s -check-prefix=ATOMICS
+// RUN: %clang --target=wasm32-unknown-unknown -### %s -mno-atomics 2>&1 | FileCheck %s -check-prefix=NO-ATOMICS
+
+// ATOMICS: "-target-feature" "+atomics"
+// NO-ATOMICS: "-target-feature" "-atomics"
+// DEFAULT-NOT: "-target-feature" "-atomics"
+// MVP-NOT: "-target-feature" "+atomics"
+// GENERIC-NOT: "-target-feature" "+atomics"
```

Comment:
I think this PR was mixed with #80775 and these `GENERIC` lines were what I intended for #80775 and added here by mistake. But @tlively pointed out that we had `DEFAULT` checks so we didn't need those lines. But later I found out that these CPU tests were not working as intended anyway and plan to delete these lines (See #80900). Sorry for the confusion...